docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
+`--force`, `-f`           | `boolean` | no           | -           | Generate preview output even when errors occur
+`--help`                  | `boolean` | no           | -           | Show help
+`--port`, `-p`            | `number`  | no           | 8080        | Preview port
+`--skip-decorator`        | `array`   | no           | -           | Ignore certain decorators
+`--skip-preprocessor`     | `array`   | no           | -           | Ignore certain preprocessors
+`--use-community-edition` | `boolean` | no           | -           | Force using Redoc Community Edition for docs preview
+`--version`               | `boolean` | no           | -           | Show version number
+
+## Examples
+
+### Entrypoints
+
+The command behaves differently depending on how you pass an entrypoint to it and whether the [configuration file](#custom-configuration-file) exists. There are the following options:
 
-```shell
-Positionals:
-  entrypoint                                                 [string] [required]
+#### Pass entrypoint directly
 
-Options:
-  --version                Show version number.                        [boolean]
-  --help                   Show help.                                  [boolean]
-  --port, -p               Preview port.                [number] [default: 8080]
-  --skip-preprocessor      Ignore certain preprocessors.                 [array]
-  --skip-decorator         Ignore certain decorators.                    [array]
-  --use-community-edition  Force using Redoc CE for docs preview.      [boolean]
-  --force, -f              Produce bundle output even when errors occur.
-                                                                       [boolean]
-  --config                 Specify path to the config file.             [string]
+```bash
+openapi preview-docs openapi/openapi.yaml
 ```
 
+In this case, `preview-docs` will preview the definition that was passed to the command. The configuration file is ignored.
 
-### How to preview the docs on a custom port
+#### Pass entrypoint without extension
 
-By default, without providing a port, the preview starts on port 8080, and can be accessed at `http://localhost:8080`.
+You can omit entrypoint's file extension when executing the `preview-docs` command. In this way, you can reference either `.yaml` or `.json` file.
 
-This command starts a preview on port 8888, and you can access the docs at `http://localhost:8888` after running it.
+```bash
+# preview-docs will preview either petstore.yaml or petstore.json file in the current working directory
+openapi preview-docs petstore
+# preview-docs will preview either sandbox.yaml or sandbox.json file in the openapi/extra directory
+openapi preview-docs openapi/extra/sandbox
+```
+
+#### Pass entrypoint via configuration file
+
+Instead of a full path, you can use an alias assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. For example, `petstore`:
```

Comment:
```suggestion
Instead of a full path, you can use an alias assigned in the `apiDefinitions` section within your `.redocly.yaml` configuration file as the entrypoint. For example, `petstore`:
```